crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -709,6 +712,11 @@ impl SpecializedMeshPipeline for MeshPipeline {
         let mut shader_defs = Vec::new();
         let mut vertex_attributes = Vec::new();
 
+        shader_defs.push("INSTANCE_INDEX".into());
```

Comment:
Could this be renamed to `INSTANCE_INDEX_AS_VERTEX_OUTPUT` (or `INSTANCE_INDEX_TO_FRAGMENT`)? It would describe better why you'd want to add this define.